### DISCUSSION TITLE
2009/02/09/%E5%AF%B9%E8%BF%91%E6%9C%9F%E8%82%A1%E5%B8%82%E5%92%8C%E6%A5%BC%E5%B8%82%E7%9A%84%E9%A2%84%E6%B5%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#252] [Announcements]
# 2009/02/09/%E5%AF%B9%E8%BF%91%E6%9C%9F%E8%82%A1%E5%B8%82%E5%92%8C%E6%A5%BC%E5%B8%82%E7%9A%84%E9%A2%84%E6%B5%8B

去年，可能中国大部分地区楼市都跌了。但上海房价并没有明显下降，因为上海的买房需求还是挺旺盛的。不过，我估计今年形势会有所不同。始于去年的经济危机造成了很多公司降薪甚至裁员，尤其是原本工资待遇不错的外资企业。这又导致了大多数人调低了对未来薪资的预期。如果一个人预期将来收入会稳步增长，他会对贷款消费持乐观态度。相反如果预期自己未来的收入不会有明显增加，甚至可能下降，则不敢贷款。所以，未来一年，购房需求会明显下降。而供求的不平衡必然导致房价下跌。

https://blog.qizhen.xyz/2009/02/09/%E5%AF%B9%E8%BF%91%E6%9C%9F%E8%82%A1%E5%B8%82%E5%92%8C%E6%A5%BC%E5%B8%82%E7%9A%84%E9%A2%84%E6%B5%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

jew
2009年02月18日 19:10 编辑
呵呵。楼主高见

回复
yoyo
2009年02月10日 20:25 编辑
请客了

回复
